### ISSUE TITLE
[references] Add learning rate scheduler

### ISSUE THREAD

**Issue by charlesmindee** [mindee/doctr#347]
Add learning rate scheduler on both tensorflow and pytorch scripts.

- [x] In TensorFlow ExponentialDecay (#360) 
- [ ] In Pytorch (#381)
